Source: compnerd@compnerd.org
Upstream: no
Reason: bad man8dir
--- sgml-common-0.6.3/doc/man/Makefile.am	2008-07-10 21:19:02.000000000 -0700
+++ sgml-common-0.6.3/doc/man/Makefile.am	2008-07-10 21:19:07.000000000 -0700
@@ -2,2 +2,2 @@
-man8dir	  = $(mandir)/man8
-man8_DATA = *.8
+man_eightdir	  = $(mandir)/man8/
+man_eight_DATA = *.8
